packages/webapp/src/components/Task/FieldWorkTask/index.jsx

```diff
@@ -41,6 +36,31 @@ const PureFieldWorkTask = ({
   useEffect(() => {
     if (fieldWorkTypeExposedValue?.value !== 'OTHER') setValue(FIELD_WORK_OTHER_TYPE, null);
   }, [fieldWorkTypeExposedValue]);
+
+  useEffect(() => {
+    setFieldWorkTypeOptions((allOptions) => {
+      let options = fieldWorkTypes.map((f) =>
+        f.farm_id === null ? { ...f, label: t(f.label) } : { ...f, label: f.field_work_name },
+      );
+      options.push({ label: t('ADD_TASK.FIELD_WORK_VIEW.TYPE.OTHER'), value: 'OTHER' });
+      return options;
+    });
+  }, [fieldWorkTypes]);
+
+  useEffect(() => {
+    dispatch(getFieldWorkTypes());
+  }, []);
+
+  const displayLabel = (value) => {
+    const translationKey = `ADD_TASK.FIELD_WORK_VIEW.TYPE.${value}`;
+    if (i18next.exists(translationKey)) {
+      return t(translationKey);
+    } else {
+      let customValue = value.toLocaleLowerCase().replaceAll('_', ' ');
```

Comment:
You can consider using the name supplied instead of lowercasing the translation key. Maybe the capitalization is important?:
![image](https://user-images.githubusercontent.com/30075973/204936922-6b0b1ae9-d101-4700-bfa6-d4ca90291caa.png)

![image](https://user-images.githubusercontent.com/30075973/204936966-3f6ffaa8-bb7c-4a67-837b-3f6973593a3b.png)